packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -2979,6 +2985,15 @@ class BuildOwner {
   }
 }
 
+/// Concrete implementation of [BuildOwner] for the widgets library.
```

Comment:
Do we need this class? Could the BuildOwner just be non-abstract and whoever creates it pass in a FocusManager?